docs/components/concepts/messages.md

```diff
@@ -19,13 +19,21 @@ When a message is published and the message name and correlation key match to a
 A subscription is closed when the corresponding element (e.g. the message catch event), or its scope is left. After a subscription is opened, it is not updated (for example, when the referenced process instance variable is changed.)
 
 <details>
-   <summary>Publish message via zbctl</summary>
+   <summary>Publish message via Camunda 8 REST API</summary>
    <p>
 
 ```
-zbctl publish message "Money collected" --correlationKey "order-123"
+curl -L 'http://localhost:8080/v2/messages/publication' \
+-H 'Content-Type: application/json' \
+-H 'Accept: application/json' \
+-d '{
+  "name": "Money collected",
+  "correlationKey": "order-123"
+}'
 ```
 
+See the [API reference for publish message](/apis-tools/camunda-api-rest/specifications/publish-message.api.mdx) for more information, including additional request fields and code samples.
+
```

Comment:
```suggestion
When you require immediate feedback if the message was correlated to an open subscription, you can use `Correlate message` via Camunda 8 REST API. If correlation is successful it will return the first process instance key the message correlated with.

curl -L 'http://localhost:8080/v2/messages/correlation' \
-H 'Content-Type: application/json' \
-H 'Accept: application/json' \
-d '{
  "name": "Money collected",
  "correlationKey": "order-123"
}'

See the [API reference for correlate message](/apis-tools/camunda-api-rest/specifications/correlate-message.api.mdx) for more information, including additional request fields and code samples.
```

Reply:
I like this add, but we may do some copy edits! Committing the suggestion. Thank you!